work/Abigail/excercise_1.2.py

```diff
@@ -0,0 +1,28 @@
+import os
+import subprocess
+import sys
+
+#this need to be looked over!
+def main():
+    if len(sys.argv) < 2:
+        print("Usage: python bash_terminal.py <command>")
+        sys.exit(1)
+
+    command = sys.argv[1]
+    
+    # Fork the process
+    pid = os.fork()
+    if pid < 0:
+        print("Fork failed")
+        sys.exit(1)
+    elif pid == 0:  # Child process
+        # Execute the command using exec
+        subprocess.run(command, shell=True)
```

Comment:
this fuction of subprocess do inside a fork and a exec : its better only do a exec

Reply:
In the most recent commit "final additions to code and updating to use correct command" I changed this to use exec instead